### PR TITLE
Add Contact Section and Copy to Clipboard Button Functionality

### LINKED ISSUE #13: Contact Copy to Clipboard Functionality

### PR DESCRIPTION
**Pull request by chrismekim**
Also closes #13 
- Note: Copy to clipboard button functionality is for the Contact page

Preview: 
Screen Shot 2020-09-18 at 2 34 56 PM

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -190,6 +190,37 @@ <h2 class="text-subheader color-gray pb2 sm:pb1_75 md:pb2_5">
           </div>
         </div>
       </section>
+
+      <section
+        class="contact-section flex flex-col sm:flex-row justify-between items-center site-content-max-width inner-content-max-width mxauto contact-page sm:pt4 sm:pb5 site-padding-x"
```
Comment (mokaymokay) [inline src/index.html]: the two max-width classes are contradictory here

File: src/index.html [hunk 2]
```diff
@@ -190,6 +190,37 @@ <h2 class="text-subheader color-gray pb2 sm:pb1_75 md:pb2_5">
...
+            src="img/contact-illustration-lg.png"
+            srcset="
+              img/contact-illustration-sm.png 240w,
+              img/contact-illustration-md.png 400w,
+              img/contact-illustration-lg.png 720w
+            "
+            sizes="(min-width: 1200px) 720w,
+            (max-width: 768px) 400w,
+            (max-width: 380px) 240w"
+            alt="Illustration of person inside a large microphone yelling into an ear."
```
Comment (mokaymokay) [inline src/index.html:220]: I love these alts lol

**Review by mokaymokay — APPROVED** [review]

File: src/index.html (hunk 2 above)
Comment (chrismekim) [reply src/index.html:220]: 🤣 🤣 🤣

File: src/index.html (hunk 1 above)
Comment (chrismekim) [reply src/index.html]: Gooood catch, ty!

**Review by whoisgina — APPROVED** [review]

**Comment by whoisgina** [comment]
Copy email to clipboard: the reasonable option for a contact page. 👍

**Pull request merged**